.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "restructuredtext.confPath": ""
```

Comment:
do you mind adding to .gitignore an ignore on .vscode? Is there a reason to add his to the repo?

Reply:
Oops, I'll fix that now.